content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -0,0 +1,202 @@
+# Sharing Experiments
+
+There are two types of remotes that can store experiments. Git remotes are
+distributed copies of the Git repository, for example on GitHub or GitLab.
+
+[DVC remotes](/doc/command-reference/remote) on the other hand are
+storage-specific locations (e.g. Amazon S3 or Google Drive) which we can
+configure with `dvc remote`. DVC uses them to store and fetch large files that
+don't normally fit inside Git repos.
+
+DVC needs both kinds of remotes for backing up and sharing experiments.
```

Comment:
I already shortened the intro but maybe collapse all this to a single sentence like "DVC requires both Git remotes and DVC remote storage to..." (with appropriate links)

Reply:
The next paragraph reiterates on what each remote is used for anyway.

Reply:
I guess no need hyper compact everything in the UG. It's good to have some longer explanations probably.